functions/functions/twilio/messages.protected.js

```diff
@@ -0,0 +1,106 @@
+// twilio client
```

Comment:
Could you please explain the purpose of this file? it appears to be duplicating a lot of logic & isn't clear to me the purpose in general